tests/snapshots.rs

```diff
@@ -78,10 +78,26 @@ struct Parameters {
 #[allow(unused_variables)]
 fn check_targets(module: &naga::Module, name: &str, targets: Targets) {
     let root = env!("CARGO_MANIFEST_DIR");
-    let params = match fs::read_to_string(format!("{}/{}/{}.param.ron", root, BASE_DIR_IN, name)) {
-        Ok(string) => ron::de::from_str(&string).expect("Couldn't parse param file"),
-        Err(_) => Parameters::default(),
+    let params = {
+        let params =
+            match fs::read_to_string(format!("{}/{}/{}.param.ron", root, BASE_DIR_IN, name)) {
+                Ok(string) => ron::de::from_str(&string).expect("Couldn't parse param file"),
+                Err(_) => Parameters::default(),
+            };
+
+        // Make MSL output parameters use params.bounds_check_policies.
```

Comment:
why aren't we doing the same thing as we do for SPIR-V? it passes the bounds check policies separately